contrib/local.mk.example

```diff
@@ -9,7 +9,7 @@
 
 # By default, the jemalloc family of memory allocation functions are used.
 # Uncomment the following to instead use libc memory allocation functions:
-# CMAKE_EXTRA_FLAGS="-DUSE_JEMALLOC=OFF"
+# DEPS_CMAKE_FLAGS := -DUSE_BUNDLED_JEMALLOC=OFF
```

Comment:
Assuming it was available at compile-time, right? I'm just trying to be clear so I know what to change the comment to (or perhaps, if you already know, you could just tell me what to change it to :-))